cmd/tailscale/backend.go

```diff
@@ -150,7 +150,7 @@ func newBackend(dataDir string, jvm *jni.JVM, appCtx jni.Object, store *stateSto
 		return nil, fmt.Errorf("runBackend: NewUserspaceEngine: %v", err)
 	}
 	sys.Set(engine)
-	b.logIDPublic = logID.Public().String()
```

Comment:
If you want to avoid dealing with casing and such, you could convert logIDPublic to an `atomic.Pointer` like [this](https://go.dev/play/p/E5cMWQal4wh).

Reply:
My bad, I meant convert just the logIDPublicAtomic on *App from atomoic.Value to atomic.Pointer.

Reply:
Done.